synapse/storage/prepare_database.py

```diff
@@ -25,7 +25,7 @@
 
 # Remember to update this number every time a change is made to database
 # schema files, so the users will be informed on server restarts.
-SCHEMA_VERSION = 52
+SCHEMA_VERSION = 53
```

Comment:
It's not always necessary to bump this - you can just keep adding more delta files into the current delta directory.